checker/tests/src/tests/FenumTest.java

```diff
@@ -16,7 +17,7 @@ public FenumTest(File testFile) {
     }
 
     @Parameters
-    public static Collection<Object[]> data() {
-        return testFiles("fenum", "all-systems");
+    public static Collection<Object[]> getTestFiles() {
```

Comment:
This method repeats a lot, in each test file, and only the directory names differ.
To reduce this duplication and possible source of mistakes, could we introduce a

String[] getTestDirectories()

method? Then one would only need to write

return {"fenum", "all-systems"};

(If array literals work in that context...)